spinalcordtoolbox/deepseg/models.py

```diff
@@ -360,6 +340,21 @@
                              'dataset has healthy control subjects. The model has been trained in a human-in-the-loop active learning fashion.',
          'url': 'https://github.com/sct-pipeline/fmri-segmentation',
          'models': ['model_seg_sc_epi_nnunet']},
+    'seg_ms_lesion_mp2rage':
```

Comment:
I notice that we don't have a test for this task. Presumably, this is because we don't have MP2RAGE data in our testing dataset?

However, it might be worth adding a test anyway using dummy T2 data, just to see whether it even runs (and if any errors are thrown).

This can be easily done by adding a new case to the existing `test_segment_nifti_binary_seg` test:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/ecea2beaadc67ec7c02f6b539592db3bc01a1495/testing/cli/test_sct_deepseg.py#L41-L58

The new test case would look something like:

```python
     (sct_test_path('t2', 't2.nii.gz'),  # dummy image since no MP2RAGE test data
      None,                              # no ground truth, just test if it runs
      't2_seg_deepseg.nii.gz', 
      'seg_ms_lesion_mp2rage', 
      None), 
```